pkgs/games/shticker_book_unwritten/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, rustPlatform, openssl, pkg-config, steam-run-native, lib }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "shticker_book_unwritten";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "JonathanHelianthicusDoe";
+    repo = "shticker_book_unwritten";
```

in order to simplify the code, you could avoid repeating the name by using `inherit`

---

That wont work.

```suggestion
    repo = pname;
```

would but I am not a big fan of this. Also repeating the name is not an issue.